tests/test_util.py

```diff
@@ -20,28 +20,24 @@
 import unittest
 
 from pydgraph import util
-import pydgraph
 
-from . import helper
-
-
-class TestIsString(unittest.TestCase):
-    """Tests is_string utility function."""
+class TestUtil(unittest.TestCase):
```

Comment:
why did we change the function name here?

Reply:
the class should be named after the python module (i.e. file).  This is the pattern in the other test files.  It is clear when seen in context: https://github.com/dgraph-io/pydgraph/blob/866d3173bd1c556e819910c7017cd5dd80c5c04a/tests/test_util.py#L24-L27

Reply:
We can change the name of the file. We should just avoid the word "util" everywhere we can. It has not value in its meaning.